docs/docusaurus.config.js

```diff
@@ -63,8 +63,8 @@ const config = {
           src: 'img/jspecify-logo.svg',
         },
         items: [
-          {to: '/about', label: 'About', position: 'left'},
-          {to: '/docs/user-guide', label: 'User Guide', position: 'left'},
+          {to: '/about', label: 'About Us', position: 'left'},
+          {to: '/start-here', label: 'Learn More', position: 'left'},
```

Comment:
Above you called this "Start Here". Which is it?

Reply:
Yes, I'm wildly inconsistent right now.

Now, I'm not sure it *has* to be all the same phrase everywhere, if each does feel more suited to its context. But there's no rhyme or reason atm.

"Start here" is slightly problematic because "start doing _what?_" Start running a tool?
"Learn more" is a bit odd if you get linked to that page and it's like "more than _what?_ I didn't learn _anything_ yet...."

My shrugs have shrugs

Reply:
Ok I've decided: it is called the "Start Here" page, but the central button on the landing page will be labeled "Learn More". For now. Send a change if desired.